app/models/container_image_registry.rb

```diff
@@ -4,4 +4,10 @@ class ContainerImageRegistry < ActiveRecord::Base
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   has_many :container_images, :dependent => :destroy # What about deleted registry but containers are still running
   has_many :containers, :through => :container_images
+
+  virtual_column :full_name, :type => :string
+
+  def full_name
+    port.present? ? "#{host}:#{port}" : host
```

Comment:
@zeari is there a test for this behavior already?  If not, please add one, thanks!